mandatory/12-fizz-buzz.js

```diff
@@ -0,0 +1,44 @@
+/* 
+  Write a function that accepts an array of numbers as a parameter and loops through the numbers.
+  For each number the function should print (using console.log):
+
+  - If the number is a multiple of 3, print “Fizz” instead of the number
+  - If the number is a multiple of 5, print “Buzz” instead of the number
+  - If the number is a multiple of *both* 3 and 5, print “FizzBuzz” instead of the number
+  - If the number is none of the above, just print the number
+*/
+
+function fizzBuzz(numbers) {}
+
+/* 
+===================================================
+======= TESTS - DO NOT MODIFY BELOW THIS LINE =====
+There are some Tests in this file that will help you work out if your code is working.
+To run the tests for just this one file, type `npm test -- --testPathPattern 12-fizz-buzz` into your terminal
+(Reminder: You must have run `npm install` one time before this will work!)
+===================================================
+*/
```

Comment:
Do we need additional test cases for smaller values here too?

Reply:
Sorry, I'm not sure I follow what you're asking here?

Is your question that we're not testing for example `fizzBuzz([1, 2, 3])`? If so, then I can see it both ways: the existing test has assertions for all of those values, but we'd lack good test descriptions for those. Personally I would probably add more individual test cases with explicit test descriptions but that didn't seem to be the style in other test files in the coursework.